charts/image-registry-metrics-exporter/Chart.yaml

```diff
@@ -4,8 +4,8 @@ name: image-registry-metrics-exporter
 description: A Helm chart for Image Registry Metrics Exporter (IRME)
 
 type: application
-version: 0.2.0
-appVersion: 0.1.3
+version: 0.2.2
+appVersion: v0.1.5
```

Comment:
This is not the latest app release, https://github.com/radiofrance/image-registry-metrics-exporter/releases

Is it normal ?

i misunderstand how version/appVersion is used by helm :sweat_smile: